test/plugins/features/movies/index.test.js

```diff
@@ -20,4 +27,37 @@ describe('movieserver integration', () => {
 
   });
 
+  describe('filter', () => {
+
+    beforeEach(() => {
+      return Promise.all([
```

Comment:
nit: We use [bluebird](http://bluebirdjs.com/docs/getting-started.html) promise library everywhere in our codebase. you don't have to make that change here. just an fyi!

Reply:
Cool, thanks! I wasn't sure if I should use bluebird here, or just `Promise` since I saw that it's exposed in the setup (IIRC). I'll be sure to use bluebird in our "real" codebase.